### DISCUSSION TITLE
Screen Saver like option for UI

### DISCUSSION THREAD

**Discussion by rxritalin** [music-assistant/support#2415] [Feature requests and ideas]
I have a test build of a touch screen device with a USB DAC that does nothing else but act as a UI for Music Assistant, connected to a pair of speakers. This works pretty well, but there is a feature or lack there of in the UI/UX that irks me. You have a nice now playing screen but it requires an interaction to display it. 

That might seem a bit lazy, but from a UI/UX perspective, asking the user to click something to have this screen display is not the best. 
I AM NOT requesting that the moment you click play on an item that it defaults to the now playing screen. I know most people would find that anoying as you might want to add new speakers or keep adding things to the playlist. 

What I am requesting is that if something is playing in the UI, and there is no interaction with the UI detected for 30 seconds it automatically shows the now playing screen. 

![image](https://github.com/music-assistant/hass-music-assistant/assets/3268460/6855363d-5c8d-4dca-9bf8-492428bab95f)

**Comment by OzGav** [comment]
But what if you are steaming to five different players. Which one would you open?

**Reply by rxritalin** [reply]
@OzGav  Can the UI not detect if something is playing? 
I guess on the UI for the other devices you would need to have the concept of a default speaker setup in order for the UI to detect if something is play correctly. 

Just one more reason to have the option for a default player. 
Right now my setup is a little intel N95, connected to a touch screen, and out to the DAC built into my Yamaha amp. Its currently running Windows and I have a the Music Assistant Companion app running on it. The companion app I believe already has a default player option with its built in Squeezelite client. 
While I have you, and talking about the companion app, a true fullscreen mode would be awesome.

**Reply by OzGav** [reply]
Changing the current behaviour of MA not starting with a player selected is currently being discussed internally. @arctixdev can comment about the companion app

**Reply by jonbng** [reply]
Yeah I thought about adding a fullscreen mode as well. I might have time to implement it today
